apps/03TicTacToe.js

```diff
@@ -24,23 +24,81 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+
+//  Your code here
+  if ((board[0][0] === 'X' && board[0][1] === 'X' && board[0][2] === 'X' ) || (board[0][0] === 'O' && board[0][1] === 'O' && board[0][2] === 'O')) {
+      console.log(playerTurn + " !!! WINS!!!    ...Resetting Game");
+
+        board = [
+          [' ', ' ', ' '],
+          [' ', ' ', ' '],
+          [' ', ' ', ' ']
+        ];
+return true;
+  } else
+  return false;
 }
 
 function verticalWin() {
-  // Your code here
+        if  ((board[0][0] === 'X' && board[1][0] === 'X' && board[2][0] === 'X' )
```

Comment:
What else can you compare to other than a hard coded X or O. Each time you check for a win you don't want to check X and O, you only care about the player that just put a mark down